### PR TITLE
addresses #171 (Missing img alt attributes in admin menu)

### PR DESCRIPTION
**Pull request by senlin**
Added the missing alt tags:
* `alt="ClassicPress"` for the primary one
* `alt=""` for the multisite blogs list

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/wp-includes/admin-bar.php [hunk 1]
```diff
@@ -119,7 +119,7 @@ function wp_admin_bar_wp_menu( $wp_admin_bar ) {
 	$wp_logo_menu_args = array(
 		'id'    => 'wp-logo',
 		'title' => (
-			'<img class="cp-logo" src="' . $cp_logo_src . '" />'
+			'<img class="cp-logo" src="' . $cp_logo_src . '" alt="ClassicPress" />'
```
Comment (Mte90) [inline src/wp-includes/admin-bar.php]: better write ClassicPress Logo, if is for accessibility we need to explain what it is

Comment (senlin) [reply src/wp-includes/admin-bar.php]: I changed it per the instructions in #171 by @nylen, see https://github.com/ClassicPress/ClassicPress/issues/171#issue-373755015 which has been open for 9 days

Comment (Mte90) [reply src/wp-includes/admin-bar.php]: I think that we need to add the logo term or someone that is blind cannot understand about what we are referencing as ClassicPress. Can be a logo, a website, the download icon etc.

Comment (senlin) [reply src/wp-includes/admin-bar.php]: here you go

**Pull request merged**